remote_write.go

```diff
@@ -80,14 +109,34 @@ type Sample struct {
 	Timestamp int64
 }
 
-func (r *RemoteWrite) XSample(value float64, timestamp int64) Sample {
+func (r *RemoteWrite) sample(c goja.ConstructorCall) *goja.Object {
+	rt := r.vu.Runtime()
+	call, _ := goja.AssertFunction(rt.ToValue(xsample))
```

Comment:
as far as I understand the second argument indicates whether `goja.AssertFunction()` succeeded, should this be checked?

Reply:
The only way for it to not succed is if what we provided isn't a function ... and it is ;) So this is one of those cases where the API returns an error that we know it can't have. Also the only reasonable thing to do here will be to panic, which will happen when `call` is used either way.

Reply:
ok, makes sense, thanks :)